src/exercise/server/services/itemController.js

```diff
@@ -36,12 +36,32 @@ async function deleteTask(req, res) {
         error.statusCode = 400;
         throw error;
     }
-    await itemManager.deleteTask(taskId);
+    try{
+        await itemManager.deleteTask(taskId);
+    } catch(error) {
+        res.status(500).json(error.message);
+    }
+    res.status(200).json(req.body);
```

Comment:
you are returning as a result all the request body you got,
you should return only relevant data, in request like this you don't really need to return anything beside the 200 success